src/validate.js

```diff
@@ -0,0 +1,183 @@
+let debug = require('debug')('taskcluster-lib-validate')
+let _ = require('lodash');
+let fs = require('fs');
+let path = require('path');
+let walk = require('walk')
+let yaml = require('js-yaml');
+let urljoin = require('url-join');
+let assert = require('assert');
+let Ajv = require('ajv');
+let AWS = require('aws-sdk');
```

Comment:
nit, but every where else we use `aws` is lower case :)
I probably shouldn't care... he he